### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by dragancajic**
`required: { node: '>=18.0.0' }`

`$ gatsby`

ERROR  UNKNOWN

Gatsby requires Node.js 18.0.0 or higher (you have v16.14.2).
Upgrade Node to the latest stable release: https://gatsby.dev/upgrading-node-js

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,13 +2,13 @@
 
 1. Fork the repository
 2. Clone your fork
-3. Verify that you are running Node version 10 ([NVM](https://github.com/nvm-sh/nvm) reccomended for managing node versions)
+3. Verify that you are running sufficient Node version ([NVM](https://github.com/nvm-sh/nvm) recommended for managing node versions)
```
Comment (mluukkai) [inline CONTRIBUTING.md:5]: I'd mention there that the proper version is 10...

**Review by mluukkai — CHANGES_REQUESTED** [review]

**Review by dragancajic — COMMENTED** [review]
Reverted to recommended Node version 10.

**Review by dragancajic — COMMENTED** [review]
Delete/exclude part0b.md from PR #2449

**Review by akirataguchi115 — APPROVED** [review]
LGTM!

**Review by mluukkai — CHANGES_REQUESTED** [review]
the pr deletes file part0b.md, and it obviously should not touch that

**Pull request merged**